scripts/check_format.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+diff="$(opa fmt -d lib/ validator/)"
+
+if [[ "$diff" == "" ]]; then
+  exit
+fi
+
+echo "Formating error:"
+echo "$diff"
+echo ""
+echo "Run \"make format\" to fix."
```

Comment:
nit: suggest saying run make format locally to reproduce this error

Reply:
fixed